tests/flow_test.go

```diff
@@ -2111,3 +2111,61 @@ func TestOvsMirror(t *testing.T) {
 
 	RunTest(t, test)
 }
+
+func TestSFlowCapture(t *testing.T) {
+	test := &Test{
+		setupCmds: []helper.Cmd{
+			{"ovs-vsctl add-br br-sfct", true},
+
+			{"ovs-vsctl add-port br-sfct sfct-intf1 -- set interface sfct-intf1 type=internal", true},
```

Comment:
I'm not sure to understand your test. I thought was to create what you did except that the sFlow capture could be started on the local looback using a dedicated port and registering a sFlow agent to the bridge using this (http://docs.openvswitch.org/en/latest/howto/sflow/)

So that we use ovs as a sFlow "tor switch" and we test that the sFlow probe is able to receive sFlow traffic.